jekyll.gemspec

```diff
@@ -38,4 +38,5 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency('jekyll-sass-converter', '~> 1.0')
   s.add_runtime_dependency('jekyll-watch', '~> 1.1')
   s.add_runtime_dependency("pathutil", "~> 0.9")
+  s.add_runtime_dependency('addressable', '~> 2.4')
```

Comment:
Doesn't look like it's a dependency of jekyll, but it a dependency of `jekyll-sitemap` per a Gemfile.lock of mine using both. So it is technically not used yet.